pkg/test/access.go

```diff
@@ -12,16 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package verifier
+package test
```

Comment:
`test` is sort of bad name, maybe it could be `exporter` or something else? given that the purpose of this package is to export span in memory, I'm not quite sure about this... it's up to you.

Reply:
> `test` is sort of bad name, maybe it could be `exporter` or something else? given that the purpose of this package is to export span in memory, I'm not quite sure about this... it's up to you.

Yes, I will rename it to `testaccess` like what I do in those import statement